packages/schemas/defs.json

```diff
@@ -58,6 +58,41 @@
         "meteo/meteo--detail"
       ]
     },
+    "UIRepresentationPropertyConfiguration": {
+      "description": "Extend ui configuration field with pure UserInterface representation configuration (exemple specify placeholder from standard component, use slider component to set number instead simple number input, use color picker component to set string or yet use external widget).",
+      "type": "object",
+      "additionalProperties": false,
+      "minProperties": 1,
+      "properties": {
+        "widget": {
+          "description": "Specify widget to use. Widget is an external component hold by the current BlockProvider",
+          "additionalProperties": false,
+          "required": ["name", "version"],
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "name of widget to use for this configuration field."
+            },
+            "version": {
+              "type": "string",
+              "description": "version of widget."
+            }
+          }
+        },
+        "component": {
+          "description": "allow to extend component configuration with representation aspect. Default component (choose by convention) may be force to specific component",
```

Comment:
Je reformulerais en : "Allow to extend component configuration with representation parameters (example: placeholder) or to use another component than the default one. Default components are: Checkbox for boolean, Input type text for string, input type number for numbers."

Reply:
sur la formulation, ok.

Sur la seconde partie sur le conf des composants standards, cela sera enforcé et documenté avec la PR concernant la gestion de configuration des component.